ReText/window.py

```diff
@@ -943,8 +944,40 @@ def standardPrinter(self, title):
 		printer = QPrinter(QPrinter.HighResolution)
 		printer.setDocName(title)
 		printer.setCreator('ReText %s' % app_version)
+		if globalSettings.paperSize:
+			pageSize = self.getPageSizeByName(globalSettings.paperSize)
+			if pageSize is not None:
+				printer.setPaperSize(pageSize)
+			else:
+				QMessageBox.warning(self, '',
+					self.tr('Unrecognized PaperSize setting "%s".' %
+					globalSettings.paperSize))
 		return printer
 
+	def getPageSizeByName(self, pageSizeName):
+		""" Returns a validated PageSize instance
+		corresponding to the given name. Returns
+		None if the name is not a valid PageSize.
+		"""
```

Comment:
Any reason why the docstring lines are so short?

Reply:
Fixed those so they wrap at longer length.